ios/Capacitor/Capacitor/CAPLog.swift

```diff
@@ -1,19 +1,26 @@
 public class CAPLog {
-
-    public static let config = CAPConfig()
-
+    public static var enableLogging: Bool = true
+    
+    private static var oneTimeConfigCheck: () -> () = {
+        // `dispatch_once` is not available since Swift 3. but, since static properties are implicitly lazy,
+        // this code will only execute once which is sufficient for our needs here. but since we need to do an
+        // async dispatch, there is a window of time where the default value will be valid before the config
+        // value(s) can be loaded.
+        DispatchQueue.main.async {
+            let config = CAPConfig()
+            if let configFlag = (config.getValue("ios.hideLogs") as? Bool) ?? (config.getValue("hideLogs") as? Bool) {
+                enableLogging = !configFlag
+            }
+        }
+        return {}
+    }()
```

Comment:
This is a stop-gap fix as I don't think this is really the best way to solve the problem. Ultimately, I think the `enableLogging` flag should be set externally, probably by the BridgeViewController at the point of entry, rather than letting `CAPLog` load a configuration. That will be the best way to handle things for an embedded context where the configuration might be programmatically changed. But that can addressed when `CAPConfig` is redone and the view controller is refactored.